Game/Source/GDKTestGyms/Private/TestGymsReplicationGraph.cpp

```diff
@@ -481,6 +560,45 @@ void UTestGymsReplicationGraphNode_AlwaysRelevant_ForConnection::ResetGameWorldS
 	AlwaysRelevantStreamingLevelsNeedingReplication.Empty();
 }
 
+void UTestGymsReplicationGraphNode_AlwaysRelevant_ForConnection::GatherClientInterestedActors(const FConnectionGatherActorListParameters& Params)
+{
+	InterestedActorList.Reset();
```

Comment:
`InterestedActorList.Reset(Params.Viewers.Num()*3);`
Take a greedy guess at space requirements and reserve?

Reply:
This will typically be the same from frame to frame, and reset will allow the internal array to persist its size, so don't think setting any size is necessary.

Reply:
it doesn't shrink on reset, fair dues.